src/components/Body.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import AddTaskModal from "./AddTaskModal";
+import TaskModal from "./TaskModal";
+import "../styles/Body.css";
+
+const defaultTasks = [
+  {
+    name: "Wash Laundry",
+    detail: "Wash the shirts like you wash away your tears.",
+  },
+  {
+    name: "Finish Problem Set",
+    detail: "Finish problem set like you finish your life.",
+  },
+];
+
+function Body() {
+  const [isAddModalOpen, setAddModalOpen] = useState(false);
+  const [isTaskModalOpen, setTaskModalOpen] = useState(false);
+  const [tasks, setTasks] = useState(defaultTasks);
+  const [currentTask, setCurrentTask] = useState({ name: "", detail: "" });
+
+  const addModalToggle = () => {
```

Comment:
rename to `toggleAddModal`. It's suggested that function names start with a verb